go.mod

```diff
@@ -48,9 +48,7 @@ require (
 	github.com/gravitational/configure v0.0.0-20180808141939-c3428bd84c23
 	github.com/gravitational/form v0.0.0-20151109031454-c4048f792f70
 	github.com/gravitational/kingpin v2.1.11-0.20220506065057-8b7839c62700+incompatible
-	github.com/gravitational/license v0.0.0-20210218173955-6d8fb49b117a
```

Comment:
Don't forget to revert this.

Reply:
I reverted this change and then forgot to push it 🤦 Fixed. Thanks.